notification.c

```diff
@@ -310,6 +310,33 @@ void notification_handle_button(struct mako_notification *notif, uint32_t button
 	}
 }
 
+static enum mako_touch_binding get_touch_binding(struct mako_config *config) {
+	return config->touch_bindings.press;
+}
+
+void notification_handle_touch(struct mako_notification *notif) {
```

Comment:
Yeah, this duplicates the code. Probably a good idea to unify button and touch handling.